icc-x-api/icc-contact-x-api.ts

```diff
@@ -84,9 +84,8 @@ export class IccContactXApi extends iccContactApi {
           encryptionKeys: eks.encryptionKeys
         })
 
-        let promise = Promise.resolve(contact);
-        
-        (user.autoDelegations
+        let promise = Promise.resolve(contact)
+        ;(user.autoDelegations
```

Comment:
semicolon on start line? seems formatting missed

Reply:
@whyoleg I will double check, from what I know pretty-quick is run when on commits

Reply:
Run pretty-quick, it fixed some issues but not this particular one.